repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc

```diff
@@ -0,0 +1,133 @@
+= OpenTracing Service
+
+The OpenTracing service (http://opentracing.io/) is a service in *Repose* which allows for sending tracing data to a Tracing service.
+
+The initial release of the OpenTracing service only supports using the Jaeger (http://jaeger.readthedocs.io/en/latest/) Tracer.
+However, other Tracers can be integrated in future OpenTracing service updates.
+By default, the service is disabled, but it can be enabled by including an `open-tracing.cfg.xml` configuration.
+The origin service can simply pass the Tracer-specific header to downstream services and requests will be traced across application boundaries.
+
+== General Service information
+* *Default Configuration:* open-tracing.cfg.xml
+* *Released:* vTBD
+* link:../schemas/open-tracing.xsd[Schema]
+
+== Tracer implementation detail
```

Comment:
Also maybe call this the `OpenTracing Implementation Details` or something if you agree with the reasons I gave for the previous change .

Reply:
Updated.